src/models/StyledComponent.js

```diff
@@ -181,14 +181,17 @@ export default (ComponentStyle: Function, constructWithOptions: Function) => {
       attrs,
     } = options
 
+    const styledComponentId = (options.displayName && options.componentId) ?
+      `${options.displayName}-${options.componentId}` : componentId
```

Comment:
Do we know for sure here that `componentId` will always be a thing, even if the babel plugin isn't used?